src/components/AppStart/JSSProviderShell.component.js

```diff
@@ -13,6 +13,7 @@ insertionPoint.setAttribute('id', 'jss-insertion-point');
 document.head.appendChild(insertionPoint);
 const generateClassName = createGenerateClassName();
 const jss = create(jssPreset());
+// $FlowFixMe
```

Comment:
Hey, 
Why did you add `// $FlowFixMe` when there are no changes done to `jss`? 
Thanks!